### PR TITLE
Succinct docs for run with docker

### PR DESCRIPTION
**Pull request by SBrandeis**
cc @julien-c @osanseviero

### CODE REVIEW

File: docs/hub/_toctree.yml [hunk 1]
```diff
@@ -159,6 +159,8 @@
         title: ZenML on Spaces
   - local: spaces-embed
     title: Embed your Space
+  - local: spaces-run-with-docker
```
Comment (osanseviero) [inline docs/hub/_toctree.yml:162]: Maybe also add to https://huggingface.co/docs/hub/index and https://huggingface.co/docs/hub/spaces

File: docs/hub/spaces-run-with-docker.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Run with Docker
+
+You can use Docker to run most Spaces locally.
+To view instructions to download and run Spaces' Docker images, click on the "Run with Docker" button on the top-right corner of your Space page:
+
+![Run with Docker button](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-run-with-docker.png)
```
Comment (osanseviero) [inline docs/hub/spaces-run-with-docker.md:6]: Will also need dark version

File: docs/hub/spaces-run-with-docker.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Login to the Docker registry
+
+Some Spaces will require you to login to Hugging Face's Docker registry. To do so, you'll need to provide:
+- Your Hugging Face username as `username`
+- A User Access Token as `password`. Generate one [here](https://huggingface.co/settings/tokens).
+
```
Comment (osanseviero) [inline docs/hub/spaces-run-with-docker.md:13]: Should we also add some instructions about running with secrets? (e.g. specifying them in the docker run). Maybe this should be a UI change though

**Review by osanseviero — APPROVED** [review]
Thanks a lot!

**Review by julien-c — APPROVED** [review]
nice!

**Pull request merged**